src/applications/search/containers/SearchApp.jsx

```diff
@@ -58,7 +58,14 @@ class SearchApp extends React.Component {
     // If there's data in userInput, it must have come from the address bar, so we immediately hit the API.
     const { userInput, page } = this.state;
     if (userInput) {
-      this.props.fetchSearchResults(userInput, page);
+      this.props.fetchSearchResults(userInput, page, {
+        path: document.location.pathname,
```

Comment:
I like it how you made this as a catch all argument (object) instead of having these all as arguments, much cleaner! 🔥